state/store/logic.js

```diff
@@ -1,5 +1,7 @@
 import sessionService from '../session/operations';
+import fetchTrendingMoviesLogic from '../movies/service';
```

Comment:
Please, use the same structure and naming as you have for session, it's not good practice to have multiple approaches in structure

Reply:
I understood. Fixed.